### ISSUE TITLE
Hidden miner from antivirus and work in the background.

### ISSUE THREAD

**Issue by SkyLineMMM** [WyvernTKC/cpuminer-gr-avx2#108]
Hello, is there a version that is not detected by the antivirus, and that can run in the background without visual visibility?

If even paid, at what price?

**Comment by michal-zurkowski** [comment]
Pretty much every mining software is flagged as malicious by most anti virus software.
There is --background long flag but personally i never used this functionality.

**Comment by SkyLineMMM** [comment]
Yes, I know, that's why I'm asking about hiding from antivirus, I want to mine on a working server where the NOD32 antivirus is installed. And I'll experiment with the flag, it would still be nice to hide the process in the task manager.
